config/file_test.go

```diff
@@ -207,7 +223,9 @@ func TestFileStoreGetEnivironmentOverrides(t *testing.T) {
 		path, tearDown := setupConfigFile(t, testConfig)
 		defer tearDown()
 
-		fs, err := config.NewFileStore(path, false)
+		fsiner, err := config.NewFileStore(path, false)
```

Comment:
This should be `fsInner` here and everywhere else it's copied.

Reply:
Not really worried about it in a test file.

Reply:
Just to be clear, I was referring to the typo, not really the capitalization of `I`. To maintain a decent (not perfect) standard of code quality, I think we should avoid typos everywhere, not just in non-test code.

Reply:
I didn't see the missing `n`. 😄